core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/repository/TerminologyRepositoryInitializer.java

```diff
@@ -43,7 +43,7 @@
 	 */
 	@Override
 	public final Void execute(RepositoryContext context) {
-		CodeSystemEntry primaryCodeSystem = createPrimaryCodeSystem();
+		CodeSystem primaryCodeSystem = createPrimaryCodeSystem();
```

Comment:
Just a sidenote: we were thinking about removing the repository initializer functionality completely to support full customization of the default Code Systems in a given tooling (for example to create `SNOMEDCT-UK-EDITION` instead of `SNOMEDCT` on the `MAIN` branch).